vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -98,6 +98,7 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <PrecompiledHeader>Use</PrecompiledHeader>
+      <RuntimeTypeInfo>true</RuntimeTypeInfo>
```

Comment:
I don't think we want to enable RTTI

Reply:
Made fabric compile time optional for now.